Doc/library/sunau.rst

```diff
@@ -63,6 +63,8 @@ The :mod:`sunau` module defines the following functions:
 
    A synonym for :func:`.open`, maintained for backwards compatibility.
 
+   .. versiondeprecated:: 3.7
```

Comment:
Oh, sorry, the correct name of this directive is `deprecated`! Or `deprecated-removed` if you have assigned the date of removing.